threat_intel/__init__.py

```diff
@@ -284,6 +261,100 @@
 }
 
 
+#### URL scan Endpoint
+
+Calls url/scan VirusTotal API endpoint.
+Submit a url or any other Python enumerable containing the URL addresses:
+
+    >>> urls = ["http://www.google.com", "http://www.yelp.com"]
+    >>> vt.post_url_report(urls)
+
+
+#### Hash report endpoint
```

Comment:
The capitalization should be consistent throughout the README. In the previous headers "endpoint" was capitalized.